src/aselo-webchat.tsx

```diff
@@ -197,10 +197,13 @@ export const initWebchat = async () => {
   currentConfig.showEmojiPicker !== false && renderEmojis(manager, currentConfig.blockedEmojis);
 
   // Add CloseButtons
-  FlexWebChat.MessageInput.Content.add(
+  FlexWebChat.MessagingCanvas.Content.add(
     <Provider store={manager.store as any} key="closechatprovider">
       <CloseChatButtons />
     </Provider>,
+    {
+      sortOrder: -2,
```

Comment:
I am curious if there is a reason `-2` was used instead of `-1`

Reply:
The -2 sortOrder placed the buttons higher, just immediately under the MainHeader. While with -1, there will be much space between the MainHeader and the buttons.

Reply:
Neat. That's good to know.